### ISSUE TITLE
Standard Fragment support

### ISSUE THREAD

**Issue by devjta** [00ec454/Ask#30]
It would be great, if the Ask.on method also accepts android.app.Fragment and not only android.support.v4.app.Fragment.

**Comment by 00ec454** [comment]
@devjta good point. Would please create a pull request? I would be happy to merge
